ci/environment-3.6.yaml

```diff
@@ -3,13 +3,13 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - black==19.10b0
+  - black==20.8b1
   - coverage
   - dask ==2.4.0
   - dask-glm >=0.2.0
   - distributed ==2.4.0
-  - flake8
-  - isort==4.3.21
+  - flake8==3.8.3
+  - isort==5.8.0
   - msgpack-python ==0.6.2
   - multipledispatch
   - mypy
```

Comment:
For reproducible/predictable CI runs, the `mypy` versions in these files should probably be pinned as well.